package/network/utils/comgt/files/ncm.json

```diff
@@ -63,5 +63,21 @@
 		},
 		"connect": "AT*ENAP=1,1",
 		"disconnect": "AT*ENAP=0"
+	},
+	"mtk2": {
+                "initialize": [
+			"AT+CFUN=1"
+		],
+                "configure": [
+			"AT+CREG?",
```

Comment:
Is this command really necessary? It shouldn't really do anything due to the "?" at the end.